mysite/urls.py

```diff
@@ -3,7 +3,11 @@
 from django.conf.urls import url, include
 from django.contrib import admin
 
+from django.contrib.auth import views
+
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^accounts/login/$', views.login, name='login'),
+    url(r'^accounts/logout/$', views.logout, name='logout', kwargs={'next_page': '/'}),
```

Comment:
why not read the next page from settings?

Reply:
what you mean ?

Reply:
@webknjaz done